src/main/java/org/folio/eusage/reports/api/EusageReportsApi.java

```diff
@@ -222,60 +239,167 @@ private void endStream(RowStream<Row> stream, RoutingContext ctx,
         });
   }
 
-  Future<Tuple> ermLookup(RoutingContext ctx, String identifier) {
+  static Tuple parseErmTitle(JsonObject resource) {
+    String identifierValue = null;
+    JsonArray identifiers = resource.getJsonArray("identifiers");
+    for (int i = 0; i < identifiers.size(); i++) {
+      JsonObject identifier = identifiers.getJsonObject(i).getJsonObject("identifier");
+      identifierValue = identifier.getString("value");
+    }
```

Comment:
Should a break been added to return the first identifier? The current code returns the last identifier.

Reply:
I don't know what's best here. So far I've only seen one identifier.

Reply:
An array is ordered, and the first entry is probably more important then following entries.

Reply:
Not used anymore, so code removed.